<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>799</width>
    <height>785</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="BtnPreviousTab">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>610</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Tabela Anterior</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BtnNextTab">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>610</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Próxima Tabela</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="ItemSearch">
    <property name="geometry">
     <rect>
      <x>485</x>
      <y>50</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="BtnSearch">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>50</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="BtnAction">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>610</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Realizar Ação</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="Itens_tabWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>751</width>
      <height>401</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::North</enum>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Rounded</enum>
    </property>
    <property name="currentIndex">
     <number>3</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Todos os itens</string>
     </attribute>
     <widget class="QTableWidget" name="Full_Table_Show">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>721</width>
        <height>351</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Unidades</string>
     </attribute>
     <widget class="QTableWidget" name="Unit_Table_Show">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>731</width>
        <height>351</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>VMWares</string>
     </attribute>
     <widget class="QTableWidget" name="VMware_Table_Show">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>721</width>
        <height>351</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Servidores</string>
     </attribute>
     <widget class="QTableWidget" name="Server_Table_Show">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>721</width>
        <height>351</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <zorder>Itens_tabWidget</zorder>
   <zorder>BtnPreviousTab</zorder>
   <zorder>BtnNextTab</zorder>
   <zorder>ItemSearch</zorder>
   <zorder>BtnSearch</zorder>
   <zorder>BtnAction</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>799</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
